tests/topotests/bgp_linkstate_topo1/r2/linkstate.json

```diff
@@ -0,0 +1,84 @@
+{
+  "routes": {
+    "Link OSPFv3 ID:0xffffffffffffffff Local{AS:4294967295 ID:4294967295 Area:4294967295 Rtr:10.10.10.11:2.2.2.2} Remote{AS:4294967295 ID:4294967295 Area:4294967295 Rtr:10.10.10.10:1.1.1.1} Link{IPv4:10.1.0.1 Neigh-IPv4:10.1.0.2 IPv6:2001::1 Neigh-IPv6:2001::2 MT:0,2}/XX": [
```

Comment:
This looks horrible, don't we need to put all this information into another JSON key? Like:

```
"routes": {
  "linkStates": [
  {
  ...
  tlv...
  },
  {
  ...
  }
  ]
}
```

Reply:
changed to:

```
    "IPv6-Prefix OSPFv3 ID:0x20 Local{AS:65001 ID:0 Area:0 Rtr:10.10.10.10} Prefix{OSPF-Route-Type:1 IPv6:12:12::12:12/128 MT:2}/XX": [
      {
        "valid": true,
        "bestpath": true,
        "pathFrom": "external",
        "linkState": {
          "nlriType": "IPv6 prefix",
          "protocol": "OSPFv3",
          "identifier": "0x20",
          "local": {
            "as": 65001,
            "identifier": 0,
            "area": 0,
            "routerID": "10.10.10.10"
          },
          "ipv6Prefix": {
            "ospfRouteType": 1,
            "ipReachability": "12:12::12:12/128",
            "mtID": " MT:2"
          }
        },
        "weight": 0,
        "origin": "IGP"
      }
    ],
```